schematics/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="grid-container">
+  <h1 class="mat-h1">Dashboard</h1>
+  <mat-grid-list cols="2" rowHeight="350px">
+    <mat-grid-tile *ngFor="let card of cards">
+      <mat-card>
+        <mat-card-header>
+          <mat-card-title>
+            {{card.title}}
+            <button mat-icon-button [matMenuTriggerFor]="menu">
+              <mat-icon aria-label="Menu icon">more_vert</mat-icon>
```

Comment:
It might be more appropriate to have the `aria-label` be on the `button`. See the examples in `menu-a11y.html`.